src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.Generated.tt

```diff
@@ -17,18 +17,21 @@
 // Note use of MethodImplOptions.NoInlining. We have tests that are failing on certain architectures when
 // AggresiveInlining is used. Confirmed on Intel i7-6600U in 64bit.
 #>
-
-using System;
 using System.Numerics;
 using System.Runtime.CompilerServices;
 
 namespace SixLabors.ImageSharp.PixelFormats.PixelBlenders
 {
-    internal static partial class PorterDuffFunctions
+    public static partial class PorterDuffFunctions
```

Comment:
Since not directly needed for #967, I wouldn't publish this now. This way we can postpone optimizing the API.